Demo/HDNotificationView/HDNotificationView.m

```diff
@@ -222,11 +246,34 @@ - (void)hideNotificationViewOnComplete:(void (^)())onComplete
 }
 - (void)notificationViewDidTap:(UIGestureRecognizer *)gesture
 {
+    [self hideNotificationView];
+    
     if (_onTouch) {
         _onTouch();
     }
 }
 
+- (void) notificationViewDidSwipe:(UIPanGestureRecognizer *)sender
```

Comment:
Should use CGRect utility functions for calculating distance, it have best performance in this case. Should use distance and vector on handling this behavior. Ex: distance > 10.0f and vector is down to up.
Thanks